packages/common/DynamoDb.js

```diff
@@ -14,9 +14,9 @@ const { RecordDoesNotExist } = require('./errors');
 // Exported functions
 
 /**
- * Call DynamoDb getItem
+ * Call DynamoDb get
  *
- * See [DynamoDb.getItem()](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html#getItem-property)
+ * See [DynamoDb.get()](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html#getItem-property)
```

Comment:
Change doc link to the docClient get.
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html#get-property